### ISSUE TITLE
Physical buttons

### ISSUE THREAD

**Issue by ahmaddxb** [joBr99/nspanel-lovelace-ui#187]
There is no indication on the screen for the physical buttons that toggle the internal relay are on or off

**Comment by joBr99** [comment]
yes, I'm not planning to add indecators like in the stock firmware, if you want to display the state of the relays add the corresponding entity from homeassistant to the config

**Comment by ahmaddxb** [comment]
Fair enough but from a glance you cant tell if they are on or off until you walk up to the device and touch the screen

**Comment by ahmaddxb** [comment]
Sonoff should have really added led to indicate the relay state

**Comment by joBr99** [comment]
You cloud add them as overrides for the weather forecast items if you really need to, what's attached to your nspanel where you need this information on the display?

**Comment by ahmaddxb** [comment]
Its used for bathroom lights and extractor fan. Which when the door is closed you cant tell if they are on or off without opening the bathroom door or the switch indicators

**Comment by ahmaddxb** [comment]
> You cloud add them as overrides for the weather forecast items if you really need to, what's attached to your nspanel where you need this information on the display?

It will look ugly but its an option

**Comment by joBr99** [comment]
Shouldn't be too bad, you can customize the name and the icon to a lightbulb and a fan

**Comment by ahmaddxb** [comment]
Yer I will give it a try. Thanks

**Comment by joBr99** [comment]
@ahmaddxb see #233
